src/main/java/seedu/address/logic/parser/AddHouseCommandParser.java

```diff
@@ -33,6 +33,9 @@
  * Parses input arguments and creates a new AddHouseCommand object
  */
 public class AddHouseCommandParser implements Parser<AddHouseCommand> {
+    public static final String MESSAGE_INVALIDHDB = "Hdb must have a block and a level!";
```

Comment:
I think for this part right, its better to be MESSAGE_INVALID_HDB, but its minor, change it when you have the time to do so